DANS-KNAW/dans-dataverse-client-lib
# DD-677: dans-dataverse-client-lib: implement DatasetApi.listFiles()

Fixes DD-677: dans-dataverse-client-lib: implement DatasetApi.listFiles()

# Description of changes
* [x] mvn license:format  
* [x] formatting
* [x] json logged by manual test does not match [FileMeta.scala](https://github.com/DANS-KNAW/dans-dataverse-scala-lib/blob/53ba44facb9f2f19ae04e881cfa07fa0aef2cca3/lib/src/main/scala/nl.knaw.dans.lib.dataverse/model/file/FileMeta.scala)

# How to test
* start `dev_archaeology-2021-10-30.box` (commit comment lies)
* run DatasetGetFiles with command line arguments: `doi:10.5072/DAR/HKGQKI 1`
* [x] DasetGetVersion should still work.

# Related PRs 
(Add links)
* 

# Notify
@DANS-KNAW/dataversedans
